Tests/SwiftCliCoreTests/ChartConstructorTest.swift

```diff
@@ -162,4 +172,99 @@ class FullChartPrinterTests: XCTestCase {
 
     }
 
-}
+    func testFullPatternRightYOInc() throws {
+        let result = ChartConstructor().makeChart(stitchArray: [["p1", "k1", "k1", "k1"],
+                                                                ["p1", "k1", "k1", "yo", "k1"],
+                                                                ["p1", "k1", "k1", "k1", "yo", "k1"]])
```

Comment:
Might be easier to read if you instantiate the stitch array and assign it to a variable outside the `makeChart` call.